src/examples/bad_code.cpp

```diff
@@ -0,0 +1,49 @@
+#include "ros/ros.h"
+#include "std_msgs/Float64.h"
+
+
+double calc_pi    (void)
```

Comment:
Not really a syntactical error, but leaving such a wide space between the function name and (void) is not ideal